grapher/barCharts/DiscreteBarChart.tsx

```diff
@@ -71,6 +71,11 @@ const labelToBarPadding = 5
 
 const LEGEND_PADDING = 25
 
+export interface Label {
+    valueString: string
+    yearString: string
```

Comment:
I think a better name would be `timeString`? Since it could be year or date (for now, probably more in the future).